data/cncf.yaml

```diff
@@ -1419,6 +1419,20 @@
       check_sets:
         - community
         - code
+- name: kube-state-metrics
+  display_name: Kube State Metrics
+  description: Add-on agent to generate and expose cluster-level metrics. 
+  category: observability
+  logo_url: https://raw.githubusercontent.com/cncf/artwork/master/projects/kubernetes/icon/color/kubernetes-icon-color.svg
+  devstats_url: https://kubestatemetrics.devstats.cncf.io/
+  accepted_at: "2023-12-15"
+  maturity: sandbox
```

Comment:
Kubernetes is graduated and kube-state-metrics is a subproject. Does this mean it should be graduated as well?